index.html

```diff
@@ -1,111 +1,579 @@
+                target="_blank"
+                rel="noopener noreferrer"
+                >Telegram</a
+              >
+              <a
+                href="https://www.LinkedIn.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >LinkedIn</a
+              >
+              <a
+                href="https://www.Instagram.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Instagram</a
+              >
             </div>
           </div>
         </div>
-      </template>
-      <!-- <script src="script.js"></script> --> <!-- uncomment script --> 
-</body>
+      </section>
+      <section class="faqs">
+        <h2>FAQs</h2>
+        <div class="details">
+          <details>
+            <h3>What are the NTFs?</h3>
+            <button type="button">+</button>
+          </details>
+          <summary>
+            Urna vitae erat et lacus, consectetur ac nulla vestibulum lobortic.
+            Nula dapibus urna volutpat vestibulum lobortic. Nulla dapibus urna
+            volutpat venenatis, risus faucibus.
+          </summary>
+          <div>
+            <h3>How do i get NTFs?</h3>
+            <button type="button">+</button>
+          </div>
+          <div>
+            <h3>How can we buy your NTFs?</h3>
+            <button type="button">+</button>
+          </div>
+          <div>
+            <h3>Who are the team behind the project?</h3>
+            <button type="button">+</button>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div>
+        <div class="footer-logo">
+          <a href="#">Binabox</a>
+          <p>We are the best way to check the rarity of NFT collection.</p>
+        </div>
+        <div class="menu">
+          <ul>
+            <li>
+              <a href="#" class="menu">Subscribe</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Home</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Blog</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Catalog</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Contact us</a>
+            </li>
+          </ul>
+          <ul>
+            <li>
+              <a href="#" class="menu">Company</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Terms Canditions</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Privacy Policy</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Whitepaper</a>
+            </li>
+            <li>
+              <a href="#" class="menu">Help Center</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div>
+        <p>BINABOX 2022 - ALL rights reserved</p>
+        <ul>
+          <a href="#">Twitter</a>
```

Comment:
The 'ul' tag contains only list items. It can not contain any other tags except the 'li' tag. Also note that external links should have the attribute target="_blank"

Reply:
done